### PR TITLE
Add variable to disable sub projects

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: CommonCPPCTest.cmake [hunk 1]
```diff
@@ -50,23 +50,24 @@ foreach(FILE ${TEST_FILES})
     list(APPEND ALL_CPP_TESTS ${PROJECT_NAME}_${NAME})
   endif()
 
-  add_executable(${PROJECT_NAME}_${NAME} ${FILE})
```
Comment (tribal-tec) [inline CommonCPPCTest.cmake]: Why did this go?

**Comment by tribal-tec** [comment]
+1 otherwise

File: CommonCPPCTest.cmake [hunk 2]
```diff
@@ -50,23 +50,24 @@ foreach(FILE ${TEST_FILES})
...
-  set_target_properties(${PROJECT_NAME}_${NAME} PROPERTIES
-    FOLDER "Tests" OUTPUT_NAME ${NAME})
-
+  set(TEST_NAME ${PROJECT_NAME}_${NAME})
   if(NOT TARGET ${NAME}) # Create target without project prefix if possible
-    add_custom_target(${NAME} DEPENDS ${PROJECT_NAME}_${NAME})
+    set(TEST_NAME ${NAME})
   endif()
 
+  add_executable(${TEST_NAME} ${FILE})
```
Comment (eile) [inline CommonCPPCTest.cmake]: here!

Comment (tribal-tec) [inline CommonCPPCTest.cmake]: Ah!

**Pull request merged**